lib/matplotlib/table.py

```diff
@@ -146,6 +147,57 @@ def set_text_props(self, **kwargs):
         self._text.update(kwargs)
 
 
+class CustomCell(Cell):
+    """
+    A subclass of Cell where the sides may be visibly toggled.
+
+    """
+
+    EDGES = 'BRTL'
+    EDGE_ALIASES = {'open':         '',
+                    'closed':       EDGES,  # default
+                    'horizontal':   'BT',
+                    'vertical':     'RL'
+                    }
+
+    def __init__(self, *args, **kwargs):
+
+        visible_edges = kwargs.pop('visible_edges')
+        Cell.__init__(self, *args, **kwargs)
+
+        if visible_edges is None:
+            self._visible_edges = self.EDGES
+        elif visible_edges in self.EDGE_ALIASES:
+            self._visible_edges = self.EDGE_ALIASES[visible_edges]
+        else:
+            for edge in visible_edges:
+                if edge not in self.EDGES:
+                    msg = 'Invalid edge param {0}, must only be one of {1} or \
+                            string of {2}.'.format(
+                                    visible_edges,
+                                    ", ".join(self.EDGE_ALIASES.keys()),
+                                    ", ".join(self.EDGES),
+                                    )
+                    raise ValueError(msg)
+            self._visible_edges = visible_edges
+
+    def get_path(self):
+        'Return a path where the edges specificed by _visible_edges are drawn'
+
+        codes = [Path.MOVETO] * 4
+        codes.append(Path.CLOSEPOLY)
```

Comment:
When all edges are visible, I think that CLOSEPOLY is what makes for a seamless boundary, with each corner rendered the same way.  If you overwrite it with LINETO, I think you will have a corner consisting of the start of the boundary and the end of the boundary, but not properly joined.  What path do you get from Rectangle?  That is what you should get from the 'closed' case here.